avldb/src/main/scala/scorex/crypto/authds/avltree/batch/VersionedLDBAVLStorage.scala

```diff
@@ -73,7 +73,7 @@ class VersionedLDBAVLStorage[D <: Digest](store: LDBVersionedStore,
         case _: ProverLeaf[D] => Seq(pair)
       }
     } else {
-      Seq()
```

Comment:
Is it mandatory to use Seq.empty here? In some places we are using Nil instead of Seq(), in some places - Seq.empty